lib/autokey/model/abstract_hotkey.py

```diff
@@ -45,7 +45,7 @@ def set_hotkey(self, modifiers, key):
         modifiers.sort()
         self.modifiers = modifiers
         self.hotKey = key
-        if key is not None and TriggerMode.HOTKEY not in self.modes:
+        if key is not None and TriggerMode.HOTKEY in self.modes:
```

Comment:
What is the intended behavior here? Currently your 'fix' duplicates the HOTKEY triggermode by adding it only if it already exists, is that intentional?

Reply:
@sebastiansam55